### ISSUE TITLE
"Docs menu" button on aio homepage is interactable, but not visible 

### ISSUE THREAD

**Issue by jelbourn** [angular/angular#44342]
On larger viewport sizes, the "Docs menu" button is hidden. However, the button is still interactable in the DOM. The result is that assistive technology will land on this button, but the navigation cursor indicator will be a tiny rectangle in the top right corner of the screen. In general, if something is interactable, it should take up some space on the screen. 

![image](https://user-images.githubusercontent.com/838736/144329632-48b399e9-1709-44ea-98dd-7d2f51f104dd.png)

**Comment by dario-piotrowicz** [comment]
@jelbourn I believe I fixed this in #45067 (sorry I didn't notice/look for this issue :cry:)

**Comment by jelbourn** [comment]
@dario-piotrowicz cool, thanks!
